recipe/meta.yaml

```diff
@@ -1,40 +1,47 @@
 {% set name = "tangled-up-in-unicode" %}
-{% set version = "0.1.0" %}
+{% set version = "0.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name | replace("-", "_") }}-{{ version }}.tar.gz
-  sha256: d07ffa62a4d9e9e0a92582f869ac3a2859456f5c9241580c7a5d9fcf2e6e6315
+  sha256: de56dbaf32de1b8c65621f97b06b7de21cba18c8e61f3988a426ffe3c40fed36
 
 build:
-  noarch: python
```

Comment:
Skip `py<36`, https://github.com/dylan-profiler/tangled-up-in-unicode/blob/a4368c2b70f55d90381085bfcdf979f89e42138d/setup.py#L26

Reply:
Since we don't build for python 3.6 and soon even for python 3.7, I would prefer out of conciseness to leave that out.
Is that ok with you?

Reply:
I added it.